addon/lib/infinity-model.js

```diff
@@ -90,6 +120,14 @@ export default ArrayProxy.extend({
    */
   totalPagesParam: 'meta.total_pages',
 
+  /**
+    Path of the "count" param in indicating
+    number of records from HTTP response
+    @type {String}
+    @default "meta.count"
+   */
+  countParam: 'meta.count',
```

Comment:
Couple Qs before we merge - 

- Is `count` representative of the total number of records in the (serverside) collection? Or just in the response?

- What happens if the user passes count AND total_pages in the `meta`?

Reply:
Good ?  So by default, the model will determine if it can load more by checking `total_pages`.  If it is falsey, then it will check `count`.  

And yea count would be the total # of records.  So the math is `currentPage < count / perPage` to determine if it can load more.